bundles/org.openhab.binding.volumio/src/main/java/org/openhab/binding/volumio/internal/VolumioHandler.java

```diff
@@ -66,10 +66,11 @@ public VolumioHandler(Thing thing) {
 
     @Override
     public void handleCommand(ChannelUID channelUID, Command command) {
+        VolumioService volumioLocal = volumio;
         logger.debug("channelUID: {}", channelUID);
 
-        if (volumio == null) {
-            logger.debug("Ignoring command {} = {} because device is offline.", channelUID.getId(), command);
+        if (volumioLocal == null) {
+            logger.debug("Volumio service was not yet initialized.");
```

Comment:
Please remove the logger and improve the `ThingStatusDetail`

Reply:
see [other comment](https://github.com/openhab/openhab-addons/pull/15379#discussion_r1288532102)

Reply:
solved